portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,44 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+      <label class="logo">KvngSnow's Portfolio</label>
+      <ul>
+          <li><a href="http://www.github.com/JalenJenkins">Github</a></li>
+          <li><a href="https://www.instagram.com/kvngsno">Instagram</a></li>
+          <li><a href="https://www.linkedin.com/in/jalen-jenkins-b75772186">Linkedin</a><li> 
+          </ul>
+
+    <div id="image"> <img src = "images/self image.jpg" alt = "A picture of myself" style="width:128px;height:128px;"></div>
+    
+    <div id="content"> 
+
+      <h1>Jalen's Portfolio</h1>
+      
+      <p>Welcome to my Portfolio!</p>
+      
+      <h2>About me</h2>
+      <p> 
+          <br> My name is Jalen Jenkins and I am currently a Computer Science Major in Cornell University's Class
+          of 2023. Although I have only been coding since August of 2019, I feel as though I have learned a
+          ton since starting. I am working here from home in Charlotte, North Carolina, and I am incredibly
+          excited to take in everything Google has to offer! <br><br>Born and raised in Charlotte, North Carolina, 
```

Comment:
Similar comment here - in general, I almost never use `br`, and only do in a couple cases where it's definitely appropriate, like a mailing address. What you have here is a new paragraph, so instead of these, close your existing paragraph and start a new one. When you've done that, I recommend putting it on a new line of code, too, so it's nicer to read the file.